CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Guide to Making Developer Contributions
+
+## Overview:
+
+A CSV with package name and version is uploaded by the user, 
+then the risk of the packages are assessed using `riskmetric`.
+[TODO this is shakey for me.... where is this function actually called... 
+I see where we write comments to the database, 
+and where we read fields FROM the database,
+but where and how are the database's risk metrics written?]
+
+The database (SQLite by default) is updated to include the user specified package 
+and its associated metrics. 
+
+These metrics are used to populate the UI of the application. 
+Each package only needs to be entered into the database once for the user
+to see its associated risk.
+
+## Package files and folder 
+
+[TODO order in which to introduce these files, 
+maybe also include a schematic of how the files are related]
+
+### Root
+- `loggit.json` [TODO: log of user collected data within the app?]
+- `setup.R` Load all libraries, install libraries that aren't included. Called in `app.R.`
+- `risk_assessment_app.db` Database connection used within `Modules/DB.R`
+- `app.R` This file contains the server and UI code for displaying the app, 
+     calling the files within the Modules, Server, UI, and Utils folders.
+
+### UI/Server
+
+**Most files are in both the UI and Server folders with the same name. If files are only in one folder, the folder name is included in the extension below:**
+
+- `UI/dashboard_screen.R` contains the UI layout of the landing page of the application
+
+- `UI/login_screen.R` Input user name and role if not found in the database.
+
+- `Server/sidebar.R` Based on selected package, allow the user to submit an overall comment,
+   or overall risk decision. Comments and decisions are added to the table `Packageinfo`
+     
+- `assessment_critera.R` On the top left of the application the user can open a 
+   help modal which describes the criteria used to assess the risk of the application.
+   The text to populate this modal is located in the `Data` directory
+
+- `uploadpackage.R` Import the packages from the user supplied CSV. 
+   TODO I'm not exactly sure how this file "knows" about prior uploaded packages - ie
+   How is `Total`, `New`, `Undescovered`, and `Duplicates` populated?
+   I also see code where this function uses `db_fun` to grab the `Packageinfo` table
+   but I don't really understand what is happening in this function...]
+   The UI and server code for viewing and downloading sample data.
+
+- `communityusage_metrics.R` Using the fields within the table `CommunityUsageMetrics`,
+   create a `Highchart` graph of package downloads and versions over the lifetime of the           package. [TODO do we need to repeat the code to plot every month? Can this be cleaned up?].
+   There is also an area to leave comments, which will be added to the`CommunityUsageMetrics` table. 
+   
+- `cum_report.R` [TODO Repeats the code of `communityusage_metrics.R`]
```

Comment:
😬 yep... They seem to be the same file with different variables names, e.g., `no_of_downloads` vs `no_of_downloads1`.